src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php

```diff
@@ -101,7 +101,7 @@ public function count()
         } elseif ($numResults == 1) {
             $result = array_values($result[0]);
 
-            return $result[0];
+            return (int) $result[0];
```

Comment:
Is this to fit in with the project's coding style? I used `(int)` as it's faster than the `intval()` function and I didn't need to use the `base` parameter.